src/view/screens/Search/Search.tsx

```diff
@@ -210,7 +210,8 @@ function useSuggestedFollowsV2(): [
 
 function SearchScreenSuggestedFollows() {
   const pal = usePalette('default')
-  const useSuggestedFollows = useGate('use_new_suggestions_endpoint')
+  const gate = useGate()
+  const useSuggestedFollows = gate('use_new_suggestions_endpoint')
     ? // Conditional hook call here is *only* OK because useGate()
       // result won't change until a remount.
```

Comment:
Nit: unused comment

Reply:
It's relevant to this:

```js
    ? // Conditional hook call here is *only* OK because useGate()
      // result won't change until a remount.
      useSuggestedFollowsV2
    : useSuggestedFollowsV1
```